test/Tests/src/Data/Vector_Spec.enso

```diff
@@ -50,7 +50,7 @@ foreign python generate_py_array = """
 foreign python generate_nested_py_array = """
     return [[1, 2, 3], [4, 5]]
 
-spec = Test.group "Vectors" <|
+type_spec name alter = Test.group name <|
```

Comment:
What is the meaning of `alter`? Can we explain it?

Reply:
See 10abb5711cce4820a81a73394533d42937310f79 and its commit message. `alter` is supposed to be a function to _"provide an alternative Vector"_ implementation.